conclave-enclave/src/main/kotlin/com/r3/conclave/enclave/internal/GramineEntryPoint.kt

```diff
@@ -20,10 +23,12 @@ object GramineEntryPoint {
     private const val EXIT_ERR = -1
 
     /** Enclave metadata, retrieved from the manifest. */
-    private val isSimulation = parseBoolean(System.getenv("CONCLAVE_IS_SIMULATION_ENCLAVE")!!)
+    private val enclaveMode = EnclaveMode.valueOf(System.getenv("CONCLAVE_ENCLAVE_MODE")!!.uppercase())
     private val isPythonEnclave = parseBoolean(System.getenv("CONCLAVE_IS_PYTHON_ENCLAVE")!!)
-    private val signingKeyMeasurement = System.getenv("CONCLAVE_SIMULATION_MRSIGNER")?.let { parseHex(it) }
+    private val isSgxDebug = parseBoolean(System.getenv("SGX_DEBUG"))
+    private val simulationMrSigner = System.getenv("CONCLAVE_SIMULATION_MRSIGNER")?.let { parseHex(it) }
     private val conclaveWorkerThreads = System.getenv("CONCLAVE_ENCLAVE_WORKER_THREADS")!!.toInt()
+    private val commonErrorMessage = "Manifest not present when initialising the enclave in $enclaveMode mode"
```

Comment:
Convert this into a function, otherwise this has unnecessarily created and stored a string